migrations/20210429024120-create-organization.js

```diff
@@ -8,24 +8,33 @@ module.exports = {
         primaryKey: true,
         type: Sequelize.INTEGER
       },
-      orgName: {
+      name: {
```

Comment:
I moved all the new fields you added from ProgramDirector back into Organization